lib/active_model_serializers/model.rb

```diff
@@ -0,0 +1,43 @@
+# ActiveModelSerializers::Model is a convenient
```

Comment:
I see the value in this, but maybe it belongs in the docs, rather than in the code?